src/components/Differential/DifferentialVolcanoPlot.jsx

```diff
@@ -269,10 +269,20 @@ class DifferentialVolcanoPlot extends React.PureComponent {
         .on('click', (item, index) => {
           d3.event.stopPropagation();
           d3.select('#tooltip').remove();
-          this.transitionZoom(item, true);
+          d3.select('#nonfiltered-elements')
+            .selectAll('circle')
+            .remove();
+          let circles = item.map(bin => bin);
+          this.renderCircles([...this.state.circles, ...circles]);
```

Comment:
It seems line 275 can be removed since it maps the objects entirely, then on line 276 ...circles can be changed to ...item

Reply:
Make sense to me, changed!